src/components/document/document.tsx

```diff
@@ -56,41 +56,45 @@ const DownloadButton = ({ onClick }: { onClick: SVGClickHandler }) => {
   );
 };
 
-const PublishButton = ({ onClick, dataTestName }: { onClick: SVGClickHandler, dataTestName?: string }) => {
-  const dataTest = dataTestName || "publish-icon";
```

Comment:
Removing the client-configured `dataTest` attribute seems likely to break some tests.

Reply:
I put it back in which helped fix some of the tests!